### PR TITLE
Update Tribuo Classification sample to Conclave 1.3

### PR DESCRIPTION
**Pull request by snedamle**
upgrade tribuo classification to 1.3.

### CODE REVIEW

File: tribuo-classification/README.md [hunk 1]
```diff
@@ -177,8 +177,14 @@ Start Client 2 and pass training data to train the model
...
 
       java -jar client/build/libs/client-all.jar --role="EVALUATE"  --constraint="S:3D71AAC9ED596F3C86241A977DAFA84B255D16D633DFF7AB88A053AE2183AD14 PROD:1 SEC:INSECURE" --url="http://localhost:8080"
-   
 
+## How to generate the reflection and serialization configuration files for the enclave
+
+Conclave uses the GraalVM Native Image JVM, in which all bytecode is compiled to native code ahead of time. However, 
+this cannot always predict all usages of the Java Reflection or the Serialization.
+Undetected usages of these dynamic features must be provided to the enclave in configuration files.
+You can find these configuration files at tribuo-classification/enclave/src/main/resources/META-INF/native-image location.
```
Comment (bon000) [inline tribuo-classification/README.md]: Let's format the location path with quotes `

File: tribuo-classification/client/build.gradle [hunk 2]
```diff
@@ -10,10 +10,8 @@ application {
...
-    implementation ("org.tribuo:tribuo-all:4.1.0@pom") {
-        transitive = true // for build.gradle (i.e., Groovy)
-        // isTransitive = true // for build.gradle.kts (i.e., Kotlin)
-    }
-    implementation "info.picocli:picocli:4.6.1"
+    implementation ("org.tribuo:tribuo-classification-core:$tribuoVersion")
+
+    implementation "info.picocli:picocli:$picocliVersion"
     implementation "com.r3.conclave:conclave-web-client:$conclaveVersion"
 }
```
Comment (bon000) [inline tribuo-classification/client/build.gradle]: Can you add a newline at the end of this file, please?

**Review by bon000 — CHANGES_REQUESTED** [review]

File: tribuo-classification/client/build.gradle (hunk 2 above)
Comment (snedamle) [reply tribuo-classification/client/build.gradle]: done

File: tribuo-classification/README.md (hunk 1 above)
Comment (snedamle) [reply tribuo-classification/README.md]: done

**Review by bon000 — APPROVED** [review]
Thanks for this!

**Pull request merged**